src/main/java/HW13/SQLCompanies.java

```diff
@@ -0,0 +1,103 @@
+package HW13;
+
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SQLCompanies {
```

Comment:
It is better to create an interface and in this interface, you should write a contract.  
And after that your class should implement it.
SQLCompanies -> CompanyRepositoryImpl (and interface CompanyRepository)

Reply:
Please create an interface and use it.